dataGraphs/thematics/projects/graphs/localContexts-project-example.json

```
@@ -0,0 +1,176 @@
+{
+  "@context": {
+    "@vocab": "https://schema.org/"
+  },
+  "@type": "Project",
+  "@id": "The URL of this JSON-LD file, if present",
+  "name": "Project Title",
+  "foundingDate": "2022-11-25",
+  "url": "https://localcontextshub.org/projects/{PROJECT_ID}]",
+  "description": "Project Description",
+  "identifier": [
+    "{PROJECT_ID}",
+    "provider_id_wr2=52dgtw"
+  ],
+  "contactPoint": {
+    "@type": "ContactPoint",
+    "name": "Community C",
+    "email": "info@example-data-repository.org",
+    "url": "https://www.example-data-repository.org/about-us",
+    "contactType": "Community contact point"
+  },
+  "ethicsPolicy": {
+    "@type": "CreativeWork",
+    "name": [
+      {
+        "@language": "en",
+        "@value": "Label/Notice Title"
+      },
+      {
+        "@language": "es",
+        "@value": "Label/Notice Title"
+      }
+    ]
+  },
+  "potentialAction": {
+    "@type": "InviteAction",
+    "agent": "Org or person making invitation",
+    "name": [
+      {
+        "@language": "{language_tag}",
+        "@value": "{Label Title}"
+      },
+      {
+        "@language": "{language_tag}",
+        "@value": "{Label Title}"
+      }
+    ],
+    "description": [
+      {
+        "@language": "{language_tag}",
+        "@value": "{Label Text}"
+      },
+      {
+        "@language": "{language_tag}",
+        "@value": "{Label Text}"
+      }
+    ],
+    "actionStatus": "ActiveActionStatus",
+    "object": {
+      "@type": "person or organization",
+      "name": "name of person or org",
+      "description": "description of person or org"
+    }
+  },
+  "subjectOf": {
+    "@type": "CorrectionComment",
+    "name": [
+      {
+        "@language": "{language_tag}",
+        "@value": "{Label Title}"
+      },
+      {
+        "@language": "{language_tag}",
+        "@value": "{Label Title}"
+      }
+    ],
+    "author": "{Author Name}",
+    "text": [
+      {
+        "@language": "{language_tag}",
+        "@value": "{Label Text}"
+      },
+      {
+        "@language": "{language_tag}",
+        "@value": "{Label Text}"
+      }
+    ],
+    "url": "{PROJECT_PAGE_URL}",
+    "image": [
+      {
+        "@type": "ImageObject",
+        "url": "{ICON_IMG}",
+        "encodingFormat": "image/png"
+      },
+      {
+        "@type": "ImageObject",
+        "url": "{ICON_SVG}",
+        "encodingFormat": "image/svg"
+      }
+    ],
+    "audio": [
+      {
+        "@type": "AudioObject",
+        "url": "{AUDIO_URL}",
+        "encodingFormat": "audio/mpeg"
+      }
+    ],
+    "dateCreated": "{date Label created}",
+    "dateModified": "{date Label modified}"
+  },
+  "publishingPrinciples": [
+    {
+      "@type": "CreativeWork",
+      "name": [
```

Added Notice Public Page for verification purposes.
```suggestion
      "identifier": {
           "{Notice ID}",
           "https://localcontextshub.org/notices/{Notice ID}"
      }
      "name": [
```

---

As above, I'll commit this and modify it to the expanded PropertyValue form